CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Rename `Slice` to `Region` to simplify differentiation between Wasm memory
   region and serde's `from_slice`
+- Remove unused `cosmwasm::Error::ValidationErr`.
```

Comment:
This is designed for use by contracts. Also see `invalid` message.
In particular, I want to update the erc20 contract to use such messages and standardize the validation step